bycycle/features/burst.py

```diff
@@ -127,15 +128,15 @@ def compute_amp_fraction(df_shape_features):
     return df_shape_features['volt_amp'].rank() / len(df_shape_features)
 
 
-def compute_amp_consistency(df_shape_features):
+def compute_amp_consistency(df_shape_features, direction=None):
     """Compute amplitude consistency for each cycle.
 
     Parameters
     ----------
     df_shape_features : pandas.DataFrame
         Shape features for each cycle, determined using :func:`~.compute_shape_features`.
-    df_samples : pandas.DataFrame
-        Indices of cyclepoints returned from :func:`~.compute_cyclepoints`.
+    direction : {'next', 'last'}
```

Comment:
nit: I'd prefer stylistically if the default was the same datatype as the defined options, i.e. {'next', 'last', 'both'}.

Reply:
I think there should also be a check that one of these values was inputted (and so, for example, an error is raised if someone misspells `next` as `nxet`, which would currently be treated as if it was `None`

Reply:
I like this idea. I'll set this to default to 'both'.

I wrote an updated check func in neurodsp/utils/checks.py (unmerged in open PR) that does exactly your second comment. That could be imported here. I need to do a sweep of the whole code base, because there are probably other places type checking could be helpful.